CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,162 @@
+
+**All reports will be kept confidential.**
+
+In some cases, Quansight may determine that a public statement will need to be made. If that’s the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. Quansight commits to maintaining confidentiality with regard to the reporter of an incident.
+
+For possibly unintentional breaches of the Code of Conduct, you may want to respond to the person and point out this Code of Conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue to the maintainer of the repository in question or to the Quansight Labs Co-Directors directly. Alternatively, you can reach out to the Quansight Diversity and Inclusion Committee for advice in confidence  at `coc@quansight.com`. 
+
+Be an active bystander and inform Quansight if you notice a dangerous situation or violations of this Code of Conduct, even if they seem inconsequential.
+
+### a. How to Submit a Report
+
+In the context of this Code of Conduct, a “report” means reporting any incidents 
+of Code of Conduct violations either verbally or in writing. 
+
+Quansight is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the Quansight Repository Code of Conduct, please report it in a timely manner. 
+
+**In your report please include:**
+
+  * Your contact information.
+  * Names (real, usernames or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well.
+  * Your account of what occurred, and if you believe the incident is ongoing. If there is a publicly available record (e.g. a mailing list archive or a public chat log), please include a link or attachment.
+  * Any additional information that may be helpful.
+
+All reports will be reviewed and will result in a response that is deemed necessary and appropriate to the circumstances. Where additional perspectives are needed, the team may seek insight from others with relevant expertise or experience. The confidentiality of the person reporting the incident will be kept at all times. Involved parties are never part of the review team.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately. If an individual engages in unacceptable behavior, the review team may take any action they deem appropriate, including a permanent ban from the community.
+
+### b. Conflicts of Interest
+In the event of any conflict of interest, the community member will immediately notify the Quansight.
```

Comment:
Not sure what this was mean to say. "The Quansight Diversity & Inclusion Committee"?